packages/core/src/Camera.ts

```diff
@@ -155,6 +156,10 @@ export class Camera extends Component {
   @ignoreClone
   private _isInvViewProjDirty: BoolUpdateFlag;
   @deepClone
+  private _shaderData: ShaderData = new ShaderData(ShaderDataGroup.Camera);
+  @deepClone
+  private _depthBufferParams: Vector4 = new Vector4();
```

Comment:
Variables calculated per frame do not need to be cloned

Reply:
Done.